operating/manage-compliance/managing-compliance-20.adoc

```diff
@@ -0,0 +1,27 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="managing-compliance-20"]
+= Managing compliance (2.0)
+include::modules/common-attributes.adoc[]
+:context: managing-compliance-20
+
+toc::[]
+
+:FeatureName: Compliance (2.0)
+include::snippets/technology-preview.adoc[]
+
+By navigating to the Compliance (2.0) section in the {rh-rhacs-first} portal, you can view the compliance results associated with your integration by using the compliance operator.
```

Comment:
We could provide a link with the `Compliance Operator` text so that users who are unfamiliar have the ability to learn more about it.

I also wonder if we need to call out that the scanning performed, and reported in this section currently covers infrastructure and platform compliance. It doesn't cover workload compliance.

Reply:
@dashrews78 ^^

Reply:
@agantony 

> We could provide a link with the Compliance Operator text so that users who are unfamiliar have the ability to learn more about it.

That is a good idea to give the users some more information about what the compliance operator can do.

Reply:
```suggestion
By navigating to the Compliance (2.0) section in the {rh-rhacs-first} portal, you can view the compliance results associated with your clusters by using the compliance operator.  Note: at this time Compliance(2.0) and the compliance operator only evaluate infrastructure and platform compliance.
```

Reply:
I took a shot at a suggestion there.